apps/system/js/bluetooth.js

```diff
@@ -194,6 +196,10 @@ var Bluetooth = {
     req.onsuccess = function bt_gotDefaultAdapter(evt) {
       self.defaultAdapter = req.result;
       self.initWithAdapter(self.defaultAdapter);
```

Comment:
I remember that I have commented the function name is not good enough.
`initWithAdapter` I cannot understand what the function init for.. 
How about we revise it to be `watchProfilesStatusChanged`/`listenProfilesStatusChanged`?